en/docs/manage/api-management.md

```diff
@@ -0,0 +1,54 @@
+# Manage
+
+Choreo's API Management capabilities allow you to manage your APIs through the Choreo Console. You can manage the lifecycle, consumers, and documents, secure your backends by applying rate--limiting policies through usage plans, define permissions using scopes, configure security, rate-limiting, and visibility settings, and define any properties. 
```

Comment:
Change "rate--limiting" to "rate-limiting".

Reply:
done